web/yo/app/scripts/directives/realtimestring.js

```diff
@@ -176,6 +176,14 @@ angular.module('oncokbApp')
                 $scope.setTrackSignal = function() {
                     mainUtils.updateMovingFlag(false);
                 };
+                $scope.getMutationName = function(key, oldKey, uuid){
+                    if(mainUtils.processedInReview('remove', uuid) && oldKey){
```

Comment:
Do you need to check whether `reviewmode === true` in here?

Reply:
I don't think so. This function will only be triggered in review mode.

Reply:
What if 2 curators work on the same gene?